src/com/lilithsthrone/rendering/Pattern.java

```diff
@@ -0,0 +1,111 @@
+package com.lilithsthrone.rendering;
+
+import java.io.File;
+import java.io.FilenameFilter;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Pattern {
+	
+	private static Map<String, Pattern> allPatterns;
+	
+	private String name;
+	private String SVGString; // TODO ADD THIS
```

Comment:
have you?

Reply:
Nope, that's for a future expansion to show the patterns that would be applied in a square look, but I haven't decided on where to use it and didn't add the support for it anywhere in this PR at the moment